docs/source/contributing/setup.rst

```diff
@@ -75,6 +75,8 @@ happen.
 
       npm install -g configurable-http-proxy
 
+   Question: Is this supposed to say ACCESS instead of EACCES?
+   Question: Can we recommend `conda install nodejs` instead of running sudo npm install?
```

Comment:
There's loads of ways to do this (`nvm` is often my personal choice). We could be more specific about what and why:

. `sudo` may be required to perform a system-wide install, or nodejs can be installed in a user-writeable location with e.g. `conda install nodejs`.

Reply:
Do you want me to mention `nvm` here? Or was that more of an FYI for the sake of this PR?

Reply:
Ah good idea. I'll move that up to the Sys Reqs section